README.md

```diff
@@ -86,7 +86,7 @@ data that we use to cross check and verify.
 | Tochigi    | [新型コロナウイルス感染症について](http://www.pref.tochigi.lg.jp/e04/welfare/hoken-eisei/kansen/hp/coronakensahasseijyoukyou.html) |
 | Tochigi - Utsunomiya City | [宇都宮市における新型コロナウイルス感染症の発生状況](https://www.city.utsunomiya.tochigi.jp/kurashi/kenko/kansensho/etc/1023128.html)
 |Tokushima    | [新型コロナウイルス感染症について](https://www.pref.tokushima.lg.jp/ippannokata/kenko/kansensho/5034012#25)|
-|Tokyo    |[新型コロナウイルスに関連した患者の発生](https://www.bousai.metro.tokyo.lg.jp/taisaku/saigai/1007261/1007509.html)|
+|Tokyo    |[都内の最新感染動向](https://stopcovid19.metro.tokyo.lg.jp/)|
```

Comment:
The original URL was wrong (my fault!)

But could we also link to both the one you suggested and this one:https://www.bousai.metro.tokyo.lg.jp/taisaku/saigai/1007261/index.html

The stopcovid site doesn't really link to the data we're using, in fact, I don't know where the data actually is on the dataviz site :)

Reply:
Data is in the source code:

https://github.com/tokyo-metropolitan-gov/covid19/tree/development/data


Also links out to the data here on the 「オープンでデータを入手」link, e.g. 

* Patient data: https://catalog.data.metro.tokyo.lg.jp/dataset/t000010d0000000068

Reply:
> The stopcovid site doesn't really link to the data we're using

They do have links to the primary sources under "Whats new", but not to a listing of all past official reports unfortunately.

Otherwise its just their processed data.

Reply:
Yeah, we often go for the reports and not the data as that one is more human readable when it comes to trying to decipher and put it into the spreadsheet.

I think we should include both links. (And you've done that, Thanks!!)